docs/core/configuration.md

```diff
@@ -223,9 +221,6 @@ bootstrap_peers = ""
 # Comma separated list of nodes to keep persistent connections to
 persistent_peers = ""
 
-# UPNP port forwarding
```

Comment:
Is it my impression, or this configuration flag is currently useless? What is the effect of setting it to true here?

From the remaining of the review, it appears that no change was made on `p2p` package.

Reply:
Actually yes you're right, it appears as though it's dead code. No idea where its usage was removed from the P2P layer. Another good reason to just remove it from v0.37 :slightly_smiling_face:

Reply:
But also removing the CLI command?

Reply:
If there's no UPnP functionality in Comet v0.37+ (perhaps even v0.34+, still need to check), what's the point of offering that probe command?

Reply:
Ok, you have a point. Go ahead! :)